*** testsBoth/A-04-04-M-ok.out	2025-10-01 12:01:54.000000000 +0100
--- testsBoth/A-04-04-M-ok.outhyp	2025-10-05 16:05:33.630752772 +0100
***************
*** 17,23 ****
  0 - Sair
  Escolha uma opção: 1
  Introduza o número da obra: 3
! 3 - 10 de 10 - DVD - Era uma vez na Amadora - 20 - Ficção - Fernando Fonseca - 200505550
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 17,23 ----
  0 - Sair
  Escolha uma opção: 1
  Introduza o número da obra: 3
! Dvd -Era uma vez na Amadora - 20 - FICTIONFernando Fonseca - 200505550
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
***************
*** 27,33 ****
  0 - Sair
  Escolha uma opção: 1
  Introduza o número da obra: 7
! 7 - 5 de 5 - DVD - Lumiar Selvagem - 20 - Ficção - Pedro Fonseca - 200505551
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
--- 27,33 ----
  0 - Sair
  Escolha uma opção: 1
  Introduza o número da obra: 7
! Dvd -Lumiar Selvagem - 20 - FICTIONPedro Fonseca - 200505551
  Menu de obras
  1 - Mostrar obra
  2 - Mostrar todas as obras
