*** testsBoth/A-07-07-M-ok.out	2025-10-01 12:01:55.000000000 +0100
--- testsBoth/A-07-07-M-ok.outhyp	2025-10-05 16:05:34.619653924 +0100
***************
*** 9,15 ****
  0 - Sair
  Escolha uma opção: 1
  Ficheiro a abrir: naoExistente
! Abrir ficheiro: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro naoExistente
  Menu Principal
  1 - Abrir ficheiro
  2 - Guardar ficheiro
--- 9,15 ----
  0 - Sair
  Escolha uma opção: 1
  Ficheiro a abrir: naoExistente
! Abrir ficheiro: Operação inválida: Problema ao abrir ficheiro: naoExistente (No such file or directory)
  Menu Principal
  1 - Abrir ficheiro
  2 - Guardar ficheiro
