*** testsBoth/A-07-07-M-ok.out	2025-10-01 12:01:55.000000000 +0100
--- testsBoth/A-07-07-M-ok.outhyp	2025-10-02 10:02:34.419083050 +0100
***************
*** 8,15 ****
  7 - Menu de gestão de requisições
  0 - Sair
  Escolha uma opção: 1
- Ficheiro a abrir: naoExistente
- Abrir ficheiro: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro naoExistente
  Menu Principal
  1 - Abrir ficheiro
  2 - Guardar ficheiro
--- 8,13 ----
***************
*** 19,22 ****
--- 17,22 ----
  6 - Menu de gestão de obras
  7 - Menu de gestão de requisições
  0 - Sair
+ Escolha uma opção: naoExistente
+ Número inválido!
  Escolha uma opção: 0
